app/services/api.rb

```diff
@@ -0,0 +1,12 @@
+class Api
```

Comment:
rather then calling this something generic like api, why dont we call it something more meaningful like `TrainClient` or `TrainAPI`.
Something to signify this is for communicating with the train api